pkgs/development/tools/schemacrawler-bin/default.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchurl
+, unzip
+, jre
+}:
+
+stdenv.mkDerivation rec {
+  pname = "schemacrawler-bin";
+  version = "16.20.4";
+  src = fetchurl {
+    url = "https://github.com/schemacrawler/SchemaCrawler/releases/download/v${version}/schemacrawler-${version}-bin.zip";
+    sha256 = "sha256-YozWe3JMCG+NtGZlDp2zSIV7oWaKaep0zs5Lfr/CSWU=";
+  };
+
+  buildInputs = [ unzip ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r {config,lib} $out/
+    cat >$out/bin/schemacrawler <<EOF
+    #!/bin/sh
+    ${jre}/bin/java -cp $out/lib/*:$out/config schemacrawler.Main "\$@"
+    EOF
+    chmod +x $out/bin/schemacrawler
+  '';
+
+  preferLocalBuild = true;
+
+  meta = with lib; {
+    description = "Database schema discovery and comprehension tool";
+    homepage = "https://www.schemacrawler.com/";
```

```suggestion
    homepage = "https://www.schemacrawler.com/";
    sourceProvenance = with sourceTypes; [ binaryBytecode ];
```
Would you please set `sourceProvenance` and use `pname = "schemacrawler";` instead?
https://nixos.org/manual/nixpkgs/stable/#sec-meta-sourceProvenance

---

Added, thanks!